lib/chef/resource/registry_key.rb

```diff
@@ -151,6 +151,10 @@ class RegistryKey < Chef::Resource
       }
       property :recursive, [TrueClass, FalseClass], default: false
       property :architecture, Symbol, default: :machine, equal_to: %i{machine x86_64 i386}
+      property :only_record_changes, [TrueClass, FalseClass],
+         default: true,
+         introduced: "18.7.3",
+         description: "(no-op) disabled functionality to only record registry value changes"
```

Comment:
`deprecated: true` - or if you plan to implement this at some point, check for this in requirements and throw a warning.